.gitlab-ci-deploy.yml

```diff
@@ -0,0 +1,7 @@
+deploy_branch_to_environment_map:
```

Comment:
Looks like some unrelated commits got into the branch. Could you remove those? (e.g. rebase)